data-processing-lib/ray/pyproject.toml

```diff
@@ -15,6 +15,8 @@ dependencies = [
     # These two are to fix security issues identified by quay.io
     "fastapi>=0.109.1",
     "pillow>=10.2.0",
+    "bs4>=0.0.2",
+    "transformers>=4.41.2",
```

Comment:
Why do we need these 2? They are specific for individual transforms and are included there

Reply:
I seemed to need them, but I'll take your word for it that I don't, because maybe another setup add them and I did things out of order.